packages/editor/src/components/editors/part/Part.tsx

```diff
@@ -8,16 +8,20 @@ import { PartProps } from './usePart';
 const Part = ({ part }: { part: PartProps }) => {
   const summaryItem = `${part.name}Summary`;
   const [value, setValue] = useState(summaryItem);
+  const isAccordionOpen = value !== summaryItem;
 
   return (
     <AccordionRoot value={value} onChange={newValue => setValue(newValue === '' ? summaryItem : newValue)}>
       <AccordionItem value={part.name}>
         <AccordionHeader
           control={part.reset.dirty ? { icon: IvyIcons.Undo, label: `Reset ${part.name}`, action: part.reset.action } : undefined}
           title={part.state.validations.map(val => val.message).join('\n')}
+          isAccordionOpen={isAccordionOpen}
         >
-          <span className='accordion-state' data-state={part.state.state} data-dirty={part.reset.dirty} />
-          {part.name}
+          <div className='part-header-left'>{part.name}</div>
+          <div className='part-header-right'>
+            <span className='accordion-state' data-state={part.state.state} data-dirty={part.reset.dirty} />
+          </div>
```

Comment:
why is this needed?

Reply:
I used this to ensure that the Accordion State is displayed aligned to the right.

Reply:
I mean the part-header left and right class. Isn't it just possible to do this stuff with flex and justify-content: space-between?